mp/src/game/server/neo/neo_player.cpp

```diff
@@ -1899,6 +1904,7 @@ bool CNEO_Player::Weapon_CanSwitchTo(CBaseCombatWeapon *pWeapon)
     {
         if (!GetActiveWeapon()->CanHolster())
             return false;
+		GetActiveWeapon()->StopWeaponSound(RELOAD_NPC);
```

Comment:
Bad indentation lol

Reply:
thats a github problem I think. It's supposed to be an early return if GetActiveWeapon is null

Reply:
It's a tabs vs spaces problem; line 1907 uses tabs whereas the above two lines are space-indented. Most IDEs format it automagically so it doesn't look hideous. Although it might be worth converting that whole function to tabs, for consistency.

Reply:
ill add some brackets around it too